### DISCUSSION TITLE
How can I filter all my customers who have a birthday today?

### DISCUSSION THREAD

**Discussion by paulomenezes** [prisma/prisma#12286] [Q&A]
Hi, how can I filter all my customers who have a birthday today?

Something like this:

```
const customers = await prisma.customer.findMany({
   where: {
     birthday: {
       equals: 'YYYY-03-11',
     },
   },
});
```

Thanks in advance.

**Comment by umarov** [comment]
You might need to use a range with start of day and end of day. Something similar to this example https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#examples-41

**Reply by marcoarruda** [reply]
In the birthday example, I don't think the range query would work. The year of the birth does not matter and can't be used as criteria.

E.g: Both `1985 Jan, 10th` and `1990 Jan, 15th` should be included in the same range, because they are on January, but different years.

Let me know if I'm missing something. How would write the query in that case?

**Comment by mithatercan** [comment]
Did you solve your problem? I also want to get data between months and days without years.

**Reply by paulomenezes** [reply]
Yes, but with a raw query. This is my function:

```typescript
  public async getAllBirthday(): Promise<Customer[]> {
    const date = new Date();
    const month = date.getMonth() + 1;
    const day = date.getDate();

    return this.prisma
      .$queryRaw`SELECT * FROM "Customer" WHERE EXTRACT(MONTH FROM "birthday") = ${month} AND EXTRACT(DAY FROM "birthday") = ${day} AND "status" = 'ACTIVE'`;
  }
```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
